src/go/intelligence/int_test/test_config.json

```diff
@@ -66,8 +66,19 @@
       },
       "annotationMap": {
         "label": "shortDescription",
```

Comment:
Does the label should be applied to all annotations or individual annotations may have individual labels?

Reply:
here the label selectively chooses field in the alert data where all information about the alert is defined by the operators. Then predefined keywords are matched in this descriptive field. So I think it is not necessary to put it into individual annotations. Because operators use one field to put details in (eg. for SSB its "shortDescription, for GGUS its "Subject")